ecs-cli/modules/cli/local/up_app.go

```diff
@@ -46,15 +47,19 @@ func Up(c *cli.Context) {
 	if err := options.ValidateFlagPairs(c); err != nil {
 		logrus.Fatalf(err.Error())
 	}
-	composePath, err := composeProjectPath(c)
+	basePath, err := composeProjectPath(c)
 	if err != nil {
-		logrus.Fatalf("Failed to create Compose file due to:\n%v", err)
+		logrus.Fatalf("Failed to create Compose files due to:\n%v", err)
+	}
+	overridePaths, err := composeOverridePaths(basePath, c.StringSlice(flags.AddOverrides))
```

Comment:
Ah, so all the override files are added as an array of strings to the one flag? For some reason I thought we were going to have each override file specified with the same override flag.

Reply:
> For some reason I thought we were going to have each override file specified with the same override flag.

Yeah, that's how it works for StringSlice flags. For example, a user can add multiple override files like this: `local up -a override1.yml -a override2.yml`. The resulting slice will have two elements `{"override1.yml", "override2.yml"}`